nominal/cli/auth.py

```diff
@@ -1,8 +1,12 @@
 from __future__ import annotations
 
+import warnings
+
 import click
+import requests
```

Comment:
unused

Reply:
will be fixed with https://github.com/nominal-io/nominal-client/pull/96 !